### DISCUSSION TITLE
pullしてもREADME.mdしか取り込めない

### DISCUSSION THREAD

**Discussion by matsutakechan2go** [apprenticeteam2/task#3] [General]
コードのtaskをcloneしてもpullしてもREADME.mdしか取り込めない。

git clone --recurse-submodules [リポジトリのURL]
というようなコマンドも試したが変わらず取り込めなかった

**Comment by jagaimoUK** [comment]
隠しファイルの存在を忘れていたため、 `readme.md` を取り込めていないと勘違いしていた。

**Comment by jagaimoUK** [comment]
# 問題点
ローカルにプルをする際、リモートのメインブランチからプルをしている

# 解決策
１．ローカルでブランチを作成
` git checkout -b docker `                                                              
２．リモートのブランチを指定して、特定のローカルブランチにプル
下の場合、リモートの`docker`ブランチを、ローカルの`dockerブランチ`にプル
`git pull origin docker:docker`
